### PR TITLE
new icon: angularmaterial (original, plain)

### LINKED ISSUE #1736: [ICON REQUEST]: Angular Material
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### About the icon

Angular Material Design is the most commonly used component library by Angular developers

### Links and sources

Website : https://material.angular.io/

Logo and Resources : https://angular.io/presskit#angular-material

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by kilianpaquier**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes NONE**

Closes #1736

## Link to prove your SVG is correct and up-to-date.

https://angular.io/presskit#angular-material

### CODE REVIEW

**Review by Panquesito7 — DISMISSED** [review]
Looks great. Thanks! 🚀

image
image

File: devicon.json [hunk 1]
```diff
@@ -345,6 +345,24 @@
...
+    {
+        "name": "angularmaterial",
+        "altnames": [],
+        "tags": [
+            "framework",
+            "javascript"
+        ],
+        "versions": {
+            "svg": [
+                "original"
```
Comment (Snailedlt) [inline devicon.json:357]: Plain can also be added here :)
```suggestion
                "original",
                "plain"
```

**Review by Snailedlt — APPROVED** [review]
I added the `plain` icon to versions.svg in `devicon.json` (hope you don't mind) :)

The PR looks excellent! 💯 
Thank you for another great PR @kilianpaquier 🚀

**Review by Panquesito7 — APPROVED** [review]

**Comment by lunatic-fox** [comment]
⚠ Not a big issue, but icons are without a newline at the end.

**Comment by Snailedlt** [comment]
@lunatic-fox Yeah, not a big issue :)

**PS:** please check discord, and don't merge any PR's today due to release attempt

**Pull request merged**